probe/ebpf/flow.c

```diff
@@ -28,15 +28,20 @@
 #include "bpf.h"
 #include "flow.h"
 
-#define DEBUG 1
-
 // Fowler/Noll/Vo hash
 #define FNV_BASIS ((__u64)14695981039346656037U)
 #define FNV_PRIME ((__u64)1099511628211U)
 
 #define IP_MF     0x2000
 #define IP_OFFSET	0x1FFF
 
+#define MAX_VLAN_LAYERS 5
+struct vlan {
+	__u16		tci;
```

Comment:
in "classic" flow we do have the vlanid as part of the network. I think we should report it to have the same behavior

Reply:
in the "classic" way we just report/calculate the ID on the Link layer.